docs/sources/tasks/migrate/from-static.md

```diff
@@ -197,12 +217,12 @@ prometheus.scrape "metrics_test_local_agent" {
 
 prometheus.remote_write "metrics_test" {
     endpoint {
-        name = "test-3a2a1b"
+        name = "test-4dec64"
         url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
 
         basic_auth {
-            username = "<USERNAME>"
-            password = "<PASSWORD>"
+            username = "USERNAME"
+            password = "PASSWORD"
```

Comment:
```suggestion
            username = "<USERNAME>"
            password = "<PASSWORD>"
```

We should keep the < and > for variables... and define them under the code block if necessary as well.